src/components/shared/Datatable.vue

```diff
@@ -110,6 +110,18 @@
                     return data
                 },
             },
+
+            /**
+             * getSortParam function for altering vuetables default sort query string
+             *
+             * The default for this prop works with core
+             */
+            getSortParam: {
+                type: Function,
+                default(sortOrder) {
+                    return sortOrder.map(sort => (`${sort.sortField}, ${sort.direction}`))
```

Comment:
Also, I think the space between the comma and sort direction is adding a '+' to the query param which I don't is think handled by core.
```js
return sortOrder.map(sort => (`${sort.sortField},${sort.direction}`)).join('')
```

Reply:
I didnt notice a + being parsed in chrome but i removed it anyway incase we have any issues with other browsers etc

Reply:
I'm using chrome and it was adding it for me, even if it sent it with a space Core wouldn't have parsed it correctly, `' desc' !== 'desc'`